content/en/dora_metrics/data_collected/_index.md

```diff
@@ -63,7 +63,10 @@ Datadog breaks down change lead time into the following metrics, which represent
 
 These metrics are only computed when the source of the repository metadata is GitHub, and there must be a pull request (PR) associated with a commit, if any. A commit is associated with a PR if the commit is first introduced to the target branch when merging that PR. If a commit does not have an associated PR, only `dora.time_to_deploy` and `dora.deploy_time` metrics are available.
 
-**Note:** These metrics are emitted for every commit and not per deployment.
+**Notes:**
+
+- These metrics are emitted for every commit and not per deployment.
+- Make sure to familiarize with the [DORA metrics limitations][12].
```

Comment:
@gturbat and I chatted offline and think it's better to just add a link to the limitations from here to avoid the two identical blocks of text getting out of sync. @brett0000FF what do you think?